### DISCUSSION TITLE
Add eslinter to existing project

### DISCUSSION THREAD

**Discussion by Richard-Walter** [quasarframework/quasar#13016] [General - Components / Directives / etc] [answered]
Hey guys, was wondering how to add eslinter to an existing quasar project?  I didnt choose the eslinter option upon intial project creation but now would like to use it.  Thanks

**Comment by MilosPaunovic** [comment]
[https://quasar.dev/quasar-cli-webpack/linter](https://quasar.dev/quasar-cli-webpack/linter)

**Comment by rstoenescu** [comment, marked as answer]
It's easier to spawn a new project with `yarn create quasar`, pick linter (and which flavour you want), and see what packages are mentioned in package.json. Then edit your quasar.config.js and add:

```
// quasar.config.js
eslint: {
  warnings: true,
  errors: true
}
```
